tests/test_frame.py

```diff
@@ -1915,3 +1915,8 @@ def f(s: pd.DataFrame) -> pd.Series:
         return s.mean()
 
     check(assert_type(resampler.apply(f), Union[pd.Series, pd.DataFrame]), pd.DataFrame)
+    
+    
+def df_accepting_dicts_iterator() -> None: # GH 392
```

Comment:
Please put this comment on its own line (see elsewhere for examples of this)

You also have extra spaces on the two lines above.  Make sure you run `black` on your files before you commit.

Reply:
Done and ran black